README.md

```diff
@@ -0,0 +1,136 @@
+Please read the documentation available here:
+<http://mapfish.github.io/mapfish-print/>
+
+For support or questions post on the mailing list:
+<https://groups.google.com/forum/#!forum/mapfish-print-users>
+
+Prerequisites
+=============
+
+Building mapfish-print requires java 8 and gradle 5.5.1.
+
+Build
+=====
+
+Execute the following command():
+
+``` {.sourceCode .}
+> ./gradlew build
+```
+
+This will build three artifacts: print-servlet-xxx.war, print-lib.jar,
+print-standalone.jar
+
+The build also builds the documentation in the docs/build/site folder.
+To deploy the documentation it should simply be copied to the gh-pages
+branch and then committed GitHub will automatically build the updated
+site at: <http://mapfish.github.io/mapfish-print/>
+
+If you only want to build the docs simply run
+
+``` {.sourceCode .}
+> ./gradlew docs:build
+```
+
+or run build in the docs directory
+
+<div class="admonition note">
+
+The test (part of the build) requires the 'Liberation Sans' font, witch
```

Comment:
which

Reply:
I didn't check the doc for typos but it is an opportunity to go through these files. I will try to make a full pass.